run/helloworld/Dockerfile

```
@@ -28,7 +28,7 @@ WORKDIR $APP_HOME
 COPY . ./
 
 # Install production dependencies.
-RUN pip install Flask gunicorn
+RUN pip install -r requirements.txt
```

```suggestion
RUN pip install --no-cache-dir -r requirements.txt
```

Using `--no-cache-dir` in Docker speeds things up 😄